scripts/ci/lib.sh

```diff
@@ -1102,6 +1102,29 @@ send_slack_notice_for_failures_on_merge() {
     curl -XPOST -d @- -H 'Content-Type: application/json' "$webhook_url"
 }
 
+save_junit_failure() {
```

Comment:
where do we use `save_junit_failure()` other than in `deploy_stackrox()`

Reply:
nowhere as this is a new function.